charts/cruise-control/Chart.yaml

```diff
@@ -8,7 +8,7 @@ maintainers:
     url: https://ialejandro.rocks
 sources:
   - https://github.com/linkedin/cruise-control
-version: 1.0.0
+version: 1.1.4
```

Comment:
Don't change version, we replace directly on CI/CD with the new version: https://github.com/devops-ia/helm-cruise-control/blob/main/.github/workflows/release.yaml#L41-L46

Reply:
No problem - fixed.